plugins/search-backend/src/service/router.ts

```diff
@@ -169,12 +181,10 @@ export async function createRouter(
       }`,
     );
 
-    const token = getBearerTokenFromAuthorizationHeader(
-      req.header('authorization'),
-    );
-
     try {
-      const resultSet = await engine?.query(query, { token });
+      const resultSet = await engine?.query(query, {
+        credentials: await httpAuth.credentials(req),
```

Comment:
This is probably just me now being too familiar with search, but I wonder if there is a distinction between a service token and a user token at this point. I.e. is the intention of this endpoint to only serve end-user search queries or both service and end-user calls and thereby warrant the use of the `allow` option?

Reply:
Good question 🙂 
At least at Spotify, we serve both end-user and service requests... My initial understanding is that the default value allows both service and user requests (speaking of the compat http auth service), but let me double-check with @Rugvip and @freben .